### DISCUSSION TITLE
active on table tr doesn't work in v5

### DISCUSSION THREAD

**Discussion by kuchta** [saadeghi/daisyui#3323] [Q&A] [answered]
This used to work in v4 but it doesn't in v5 and currently I can't even find it in documentation...

**Comment by saadeghi** [comment, marked as answer]
`active ` class name was deprecated after daisyUI 3 (last year). And it was removed from docs since last year, in favor of using any `bg-*` class instead.
I didn't remove it immediately in daisyUI 4 (last year) because of backward compatibility.

Basically in daisyUI 3, the style was more complicated and we couldn't simply use `bg-*` class names.
It got simpler in daisyUI 4 (last year release), so instead of a new class name, we can just use `bg-base-200` to get the same result (or any other color)

v4 docs: https://daisyui.com/components/table/#table-with-an-active-row
v5 docs: https://v5.daisyui.com/components/table/#table-with-an-active-row

Let me know if you have a question.

**Reply by kuchta** [reply]
Thank you @saadeghi 🙏🏻
